pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -839,6 +895,17 @@ func aceToRules(aclArgs types.AppNetworkACLArgs, ace types.ACE) (types.IPTablesR
 	aclRule3.Action = inActions
 	aclRule3.IsUserConfigured = true
 	aclRule3.RuleID = ace.RuleID
+	if aclArgs.NIType == types.NetworkInstanceTypeSwitch {
+		if len(aclArgs.UpLinks) > 1 {
+			errStr := fmt.Sprintf("aceToRules: Switch network instance with more than ONE " +
+				"uplink attached is not supported now.")
+			log.Errorln(errStr)
+			return nil, errors.New(errStr)
+		} else {
+			aclRule3.Rule = append(aclRule3.Rule, "-m", "physdev",
+				"--physdev-in", aclArgs.UpLinks[0])
```

Comment:
Is there a reason we can't have the physdev match on aclRule3 for all cases? I guess the multiple uplinks e.g., for mesh is one reason.

Reply:
I think we can do that. I'll make the changes required.